Dockerfile

```
@@ -1,4 +1,4 @@
-FROM elixir:1.13.4-alpine AS build
+FROM hexpm/elixir:1.13.4-erlang-24.3.4.7-alpine-3.16.3 AS build
```

Our repo (`.tools-versions`) uses Erlang 25. Can you use that image instead 
```suggestion
FROM hexpm/elixir:1.13.4-erlang-25.2.3-alpine-3.16.3 AS build
```